src/ORM/Connect/PDOQuery.php

```diff
@@ -18,13 +47,70 @@ class PDOQuery extends Query
 
     protected $results = null;
 
+    /**
+     * @var string
+     */
+    protected $fetchMedthod = self::FETCH_METHOD_ALL;
```

Comment:
Typo 😉 

```suggestion
    protected $fetchMethod = self::FETCH_METHOD_ALL;
```

Obviously don't accept the suggestion without updating all the other bits of code.